R/utils.R

```diff
@@ -351,6 +351,46 @@
     altExps(x) <- values
     return(x)
 }
+
+# This function can be used to add tree to existing tree
+.add_tree <- function(x, tree, MARGIN, name = "phylo", ...) {
+  
+  MARGIN <- .check_MARGIN(MARGIN)
+  
+  if (MARGIN == 1L) {
```

Comment:
Tidier way is to get function like this https://github.com/microbiome/mia/blob/b627edce620807af20d3ed85c6667f4b8ef8f2ea/R/agglomerate.R#L452